frontend/server/src/test/resources/config.properties

```diff
@@ -1,5 +1,6 @@
 # debug=true
-# port=8080
+# inference_address=0.0.0.0:8080
```

Comment:
change to http://127.0.0.1:8080 as well (This was suggested during security review)